docs/guides-publish.md

```diff
@@ -35,104 +37,98 @@ This app has 3 environments defined:
-
 > **Note**<br>
-> The `default` environment which points to `localhost` does not have an ENS Registry address specified because aragonCLI will default the value to `0xB9462EF3441346dBc6E49236Edbb0dF207db09B7` (the ENS Registry pre-deployed on the local development chain).
+> The `default` environment which points to `localhost` does not have an ENS Registry address specified because the Buidler plugin will default the value to `0xB9462EF3441346dBc6E49236Edbb0dF207db09B7` (the ENS Registry pre-deployed on the local development chain).
 
 ## Publish a major version: content + contract
 
-To publish on aragonPM we will use [`aragon apm publish`](/docs/cli-apm-commands#aragon-apm-publish) command.
-
-Command:
+To publish on aragonPM we will use the following buidler task:
 
 ```sh
-npx aragon apm publish major --environment rinkeby
+npx buidler publish major --network rinkeby
 ```
 
 This will:
 
-<span>![*](/docs/assets/check.svg) Apply version bump (major).</span>
-
-<span>![*](/docs/assets/check.svg) _Compile_ and _deploy_ the app's contract (by default the output lives in `build`)</span>
-
-<span>![*](/docs/assets/check.svg) _Build_ the app's [frontend (by default the output lives in `dist`)](#building-frontends)</span>
-
-<span>![*](/docs/assets/check.svg) Generate application artifact.</span>
-
-<span>![*](/docs/assets/check.svg) Publish the app to the **rinkeby** environment.</span>
+- Apply version bump (major).
+- _Compile_ and _deploy_ the app's contract (by default the output lives in `build`)
+- _Build_ the app's [frontend (by default the output lives in `dist`)](#building-frontends)
+- Generate application artifact.
+- Publish the app to the **rinkeby** environment.
+- Upload the app's frontend to the IPFS server.
 
 Sample output:
 
 ```sh
- > npx aragon apm publish major --environment rinkeby
-
- ✔ Successfully published app.open.aragonpm.eth v1.0.0:
- ℹ Contract address: 0xE636bcA5B95e94F749F63E322a04DB59362299F1
- ℹ Content (ipfs): QmR695Wu5KrHNec7pRP3kPvwYihABDAyVYdX5D5vwLgxCn
- ℹ Transaction hash: 0x3d752db29cc106e9ff98b260a90615921eb32471425a29ead8cbb830fb224d8
+main     |   App name:          app.aragonpm.eth
+main     |   Initial version:   1.0.0
+main     |   Manager address:   0x5Ddb5ec4fF143fDaBCCD0a47F30FF2ce319C2a01
+main     |   Contract address:  0x2e25c8F88c5cCcbC9400e5bc86cF9C58C7604327
+main     |   ContentURI:        QmTBapuxxzFHzxdbxxZUmdEmyRxQohxoK7qiXiJ6id36tu
+main     | 
+main     |   http://localhost:8080/ipfs/QmTBapuxxzFHzxdbxxZUmdEmyRxQohxoK7qiXiJ6id36tu
+main     | 
+main     | 
+main     |   Tx sent
+main     | 
+main     |   Tx hash:  0x954fa737215152d2c9be5893bcda91cf889b30dd5e26bada10b657e87b89692f
+main     |   
+main     | 
+main     |   Tx mined
+main     |   
+main     |   Status:        Success
+main     |   Block number:  78
```

Comment:
We should probably bump this block number up to a more realistic rinkeby value :)

Reply:
Good catch hehe!